File: app/src/main/java/com/huynd/skyobserver/adapters/NavigationDrawerListAdapter.java
package com.huynd.skyobserver.adapters;

import android.content.Context;
import android.support.annotation.LayoutRes;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import com.android.annotations.NonNull;

/**
 * Created by HuyND on 8/5/2017.
 */

public class NavigationDrawerListAdapter extends ArrayAdapter<String> {
    public NavigationDrawerListAdapter(@NonNull Context context, @LayoutRes int resource,
                                       @NonNull String[] objects, @NonNull ListView listView) {
        super(context, resource, objects);
        listView.setAdapter(this);
    }
}

File: app/src/main/java/com/huynd/skyobserver/adapters/NavigationDrawerListAdapter.kt
package com.huynd.skyobserver.adapters

import android.content.Context
import android.support.annotation.LayoutRes
import android.support.annotation.NonNull
import android.widget.ArrayAdapter
import android.widget.ListView

/**
 * Created by HuyND on 8/5/2017.
 */

class NavigationDrawerListAdapter(@NonNull context: Context, @LayoutRes resource: Int,
                                  @NonNull objects: Array<String>, @NonNull listView: ListView) :
        ArrayAdapter<String>(context, resource, objects) {

    init {
        listView.adapter = this
    }
}
